Makefile

```diff
@@ -9,7 +9,7 @@ REGRESS = $(EXTENSION)
 
 DATA = pipelinedb--1.0.0.sql
 EXTRA_CLEAN = src/test/regress/expected/$(REGRESS).out src/test/regress/sql/$(REGRESS).sql
-SHLIB_LINK += /usr/lib/libzmq.a -lstdc++
+SHLIB_LINK += -lzmq -lstdc++ -fPIC
```

Comment:
found this on https://github.com/pipelinedb/pipelinedb/issues/1773#issuecomment-426924299
no idea what I'm doing

Reply:
I think this breaks the wanted static linking of libzmq in the extension, as I had to install  `zeromq-dev` in the runtime env.